src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/UsersAndRolesMigrations/20220527101017_EES3388_RenameTestUserEmails_down.sql

```diff
@@ -0,0 +1,21 @@
+UPDATE AspNetUsers SET Email = 'ees-bau1@education.gov.uk', NormalizedEmail = 'EES-BAU1@education.gov.uk', UserName = 'ees-bau1@education.gov.uk', NormalizedUserName = 'EES-BAU1@EDUCATION.GOV.UK'  WHERE UserName = 'ees-test.bau1@education.gov.uk';
+UPDATE Users SET Email = 'ees-bau1@education.gov.uk' WHERE Email = 'ees-test.bau1@education.gov.uk';
+UPDATE UserInvites SET Email = 'bau1@azurehiveitco.onmicrosoft.com' WHERE Email = 'ees-test.bau1@education.gov.uk';
+
+
+UPDATE AspNetUsers SET Email = 'ees-bau2@education.gov.uk', NormalizedEmail = 'EES-BAU2@education.gov.uk', UserName = 'ees-bau2@education.gov.uk', NormalizedUserName = 'EES-BAU2@EDUCATION.GOV.UK'  WHERE UserName = 'ees-test.bau2@education.gov.uk';
+UPDATE Users SET Email = 'ees-bau2@education.gov.uk' WHERE Email = 'ees-test.bau2@education.gov.uk';
+UPDATE UserInvites SET Email = 'bau2@azurehiveitco.onmicrosoft.com' WHERE Email = 'ees-test.bau2@education.gov.uk';
+
+UPDATE AspNetUsers SET Email = 'ees-analyst1@education.gov.uk', NormalizedEmail = 'EES-ANALYST1@EDUCATION.GOV.UK', UserName = 'ees-analyst1@education.gov.uk', NormalizedUserName = 'EES-ANALYST1@education.gov.uk'  WHERE UserName = 'ees-test.analyst1@education.gov.uk';
+UPDATE Users SET Email = 'ees-analyst1@education.gov.uk' WHERE Email = 'ees-test.analyst1@education.gov.uk';
+UPDATE UserInvites SET Email = 'analyst1@azurehiveitco.onmicrosoft.com' WHERE Email = 'ees-test.analyst1@education.gov.uk';
+
+UPDATE AspNetUsers SET Email = 'ees-analyst2@education.gov.uk', NormalizedEmail = 'EES-ANALYST2@EDUCATION.GOV.UK', UserName = 'ees-analyst2@education.gov.uk', NormalizedUserName = 'EES-ANALYST2@education.gov.uk'  WHERE UserName = 'ees-test.analyst2@education.gov.uk';
+UPDATE Users SET Email = 'ees-analyst2@education.gov.uk' WHERE Email = 'ees-test.analyst2@education.gov.uk';
+UPDATE UserInvites SET Email = 'analyst2@azurehiveitco.onmicrosoft.com' WHERE Email = 'ees-test.analyst2@education.gov.uk';
+
+
+UPDATE AspNetUsers SET Email = 'ees-analyst3@education.gov.uk', NormalizedEmail = 'EES-ANALYST3@EDUCATION.GOV.UK', UserName = 'ees-analyst3@education.gov.uk', NormalizedUserName = 'EES-ANALYST3@education.gov.uk'  WHERE UserName = 'ees-test.analyst3@education.gov.uk';
+UPDATE Users SET Email = 'ees-analyst@education.gov.uk' WHERE Email = 'ees-test.analyst3@education.gov.uk';
```

Comment:
Not super important as it's in the down method but I think there's a new mistake here in the e-mail address for Analyst 3. Shouldn't it be `ees-analyst3@education.gov.uk`?